frontend/static/js/services/authenticationService.js

```diff
@@ -1,18 +1,24 @@
 import ARequestService from "./ARequestService.js";
 import backendURL from "../constants.js";
 
+
+// TODO: Extend for tournaments
 export default class AuthenticationService extends ARequestService {
 	constructor() {
 		super();
 	}
 
-	async postRequest({username, password}) {
+	async postMatch({username, password}) {
 		return super.postRequest(
-			`${backendURL.authenticationURL}`,
+			`${backendURL.authenticationURL}match/ `, //${urlParams ? `/${urlParams}` : ''}`,
```

Comment:
I left this in, as I will need it in the upcoming PRs (either today or tomorrow sunday)